test/config/homeserver.yaml

```diff
@@ -0,0 +1,2593 @@
+# Configuration file for Synapse.
```

Comment:
Yeah, I was following richvdh's recommendation (or at least how I understood it) but it would probably be more robust if I generated the file from the default homeserver.yaml.

Do you mind if I handle this in a followup?

Reply:
I think leaving this for now makes sense! 👍